Cargo.toml

```diff
@@ -42,3 +42,5 @@ slab = "0.4.2"
 tokio = "0.1.19"
 futures = "0.1.26"
 rcgen = "0.8.14"
+openssl = "0.10.38"
```

Comment:
Can we use `rustls` instead so that we don't end up dropping Windows support? Alternatively, [`native-tls`](https://docs.rs/native-tls/) might do? Otherwise, let's add this in `target.'cfg(unix)'.dev-dependencies` instead and mark the relevant test(s) as `#[cfg(unix)]`.

Reply:
> Can we use `rustls` instead so that we don't end up dropping Windows support? Alternatively, [`native-tls`](https://docs.rs/native-tls/) might do? Otherwise, let's add this in `target.'cfg(unix)'.dev-dependencies` instead and mark the relevant test(s) as `#[cfg(unix)]`.

Sadly, neither `rustls`, nor `native-tls` support the generation of the pkcs12 format, and the mysql client library _only_ accepts pkcs12 format, so the only option I can see currently is `openssl`.  I may consider contributing to the `mysql` crate in future to allow other (better) ways to configure ssl.  Meanwhile, I will add `#[cfg(unix)]` etc.